src/store/storage/local.ts

```diff
@@ -1,17 +1,17 @@
 import { UhkConfiguration } from '../../config-serializer/config-items/UhkConfiguration';
-import { DataProviderService } from '../../services/data-provider.service';
 
 export class Local {
 
-    constructor(private dataProvider: DataProviderService) { }
+    constructor(private uhkConfiguration: UhkConfiguration) { }
 
     getConfig(): UhkConfiguration {
         let configJson = localStorage.getItem('config');
         let config: UhkConfiguration;
+
         if (configJson) {
             config = new UhkConfiguration().fromJsObject(JSON.parse(configJson));
         } else {
-            config = this.dataProvider.getUHKConfig();
+            config = this.uhkConfiguration;
```

Comment:
I think it should be. If not then we need to implement try catch and add notification about the error. User will need to refresh app to get default back. Maybe we shouldn't inject it into the local / electron, but check if `undefined` in `DataStorage` and there replace `undefined` with `uhkConfiguration`